projects/ccd-case-ui-toolkit/src/lib/shared/services/form/form-validators.service.ts

```diff
@@ -30,24 +31,42 @@ export class FormValidatorsService {
           validators.push(Validators.maxLength(caseField.field_type.max));
         }
       }
+
+      if (caseField.field_type.type === 'TextArea') {
+        validators.push(this.emptyValidator());
+        validators.push(this.markDownPatternValidator());
+      }
+
       if (control.validator) {
         validators.push(control.validator);
       }
       control.setValidators(validators);
+    } else if (caseField.display_context === 'OPTIONAL' && (caseField.field_type.type === 'Text' || caseField.field_type.type === 'TextArea')) {
+      control.setValidators(this.markDownPatternValidator());
     }
     return control;
   }
 
   public static emptyValidator(): ValidatorFn {
     const validator = (control: AbstractControl):ValidationErrors | null =>  {
-     if (control && control.value && control.value.toString().trim().length === 0) {
-        return  { required: {} };
+      if (control && control.value && control.value.toString().trim().length === 0) {
```

Comment:
why can't you use optional chain expression

Reply:
I have optimised existing code.